RAG_supporters/clustering/centroid_comparison.py

```diff
@@ -0,0 +1,471 @@
+"""
```

Comment:
Remove what is deprecated - Deprecation is not needed since those functions are not released yet